File: api/code-samples/python/klay/transaction/sendRawTransaction.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

singedTransactionData = "0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67"

sdk = OpenSDK(host)
klay_response = sdk.klay.send_raw_transaction(singedTransactionData)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/sendRawTransaction.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const singedTransactionData = '0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67'
    
    provider.eth.sendRawTransaction(singedTransactionData, {}, (err, data, response) => {
        console.log(data);
    });

}
)()